src/components/ImageEditing/ImageEditingContainer.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useState, useCallback } from "react";
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  Flex,
+  Box,
+  Button,
+  Text,
+  Input,
+  Center,
+  Slider,
+  SliderTrack,
+  SliderFilledTrack,
+  SliderThumb,
+  useToast,
+} from "@chakra-ui/react";
+import Cropper from "react-easy-crop";
+import { Point, Area } from "react-easy-crop/types";
+import { toBase64, upLoadScreenshot, inputImageCheck } from "@/utils/manageExternalImage";
+import { getCroppedImg } from "@/utils/canvasUtils";
```

Comment:
Do you think it is possible to lazy load some of these libs? Or the whole image editing container component? 

When any relatively big size component not showing on any page initially, we lazy load it,